frontend/src/lib/localStorage.js

```diff
@@ -0,0 +1,11 @@
+export function loadFromLocal(key) {
+  try {
+    return JSON.parse(localStorage.getItem(key))
+  } catch (error) {
+    console.log(error)
+  }
+}
+
+export function saveToLocal(key, value) {
+  localStorage.setItem(key, JSON.stringify(value))
+}
```

Comment:
Actually a good idea to use that two function in one file. 👍

Reply:
Thanks to the StudentCompanionApp